pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -377,6 +378,17 @@ public RpcApi convert(final String name) throws RpcApisConversionException {
   )
   private final BytesValue extraData = DEFAULT_EXTRA_DATA;
 
+  // Permissioning: A list of whitelist nodes can be passed.
+  @Option(
+    names = {"--nodes-whitelist"},
+    paramLabel = "<enode://id@host:port>",
+    description =
+        "Comma separated enode URLs for Permissioned networks. " + "Default is an empty list.",
```

Comment:
It might be just me, but 'Default is an empty list' gives me a false impression that the default behaviour is not whitelisting anyone, therefore, not talking to any node. If the user doesn't use the 'nodes-whitelist' property, we won't do any permissioning.

Do you think this can be misleading or is it just me? :P